platform/android/export/export.cpp

```diff
@@ -778,6 +778,87 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	void _write_tmp_manifest(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, bool p_debug) {
+		String manifest_header =
```

Comment:
Alternative to consider:

		String manifest_text =
				"<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
				"<manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\n"
				"    xmlns:tools=\"http://schemas.android.com/tools\">\n";

		manifest_text += _get_screen_sizes_tag(p_preset);
		manifest_text += _get_gles_tag();